packages/salesforcedx-vscode-core/src/commands/forceSourceRetrieveMetadata/forceSourceRetrieveCmp.ts

```diff
@@ -132,16 +140,92 @@ export class ForceSourceRetrieveExecutor extends SfdxCommandletExecutor<
     }
   }
 }
+export class LibraryRetrieveSourcePathExecutor extends LibraryCommandletExecutor<
```

Comment:
Do we have a test to cover this new code ?

Reply:
@lcampos No, I'll add test cases for it.

Reply:
Ok, everything else looks good btw